src/reaction_lammps_mupt/cache.py

```
@@ -0,0 +1,235 @@
+import shutil
+import subprocess
+from pathlib import Path
+from typing import Literal
+
+
+def get_git_root() -> Path:
+    """
+    Return the root directory of the current git repo.
+    Raises subprocess.CalledProcessError if not in a git repo.
+    """
+    out = subprocess.check_output(["git", "rev-parse", "--show-toplevel"], text=True).strip()
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)